tests/test_ahc.py

```diff
@@ -0,0 +1,32 @@
+"""Unit tests for the KNN classifier."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.learning.ahc import AgglomerativeHierarchicalClustering
+
+
+class TestAgglomerativeHierarchicalClustering(geomstats.tests.TestCase):
+    """Class defining the Poincare polydisk tests."""
+
+    def setUp(self):
+        """Define the parameters to test."""
+        gs.random.seed(1234)
+        self.n_clusters = 2
+        self.dimension = 2
+        self.space = Euclidean(dimension=self.dimension)
```

Comment:
Let's test this with a non-Euclidean metric, to see if it works with geomstats' distances? It'll be good to have it in the test, not only in the examples.

Reply:
I will show that it is working well with other metrics in my next PR with an example on the sphere.
Is this fine for you ?

Reply:
Ok, I will add other metrics in the test too.